### ISSUE TITLE
Tests failed

### ISSUE THREAD

**Issue by yeya** [ElishaMayer/masav#1]
Hi.

I tried to clone and run tests.
```
Test Suites: 1 failed, 1 total
Tests:       3 failed, 6 passed, 9 total
```
I think the issue is [here](https://github.com/ElishaMayer/masav/commit/cc32b9a576d42a97e4b26a916f183ea22c01d57f#diff-0bec3232bd1f8ca1e11d0f520a83d312812a3dd1fd2c38a9a49f79a64f77ba44R45).

Something is wrong in your ASCII map, why do you writing your own map?
Why do you jump from 38 to 65 (in "א" & "ב")?

**Comment by ElishaMayer** [comment]
Hi,
If it is still relevant.
For the problem, I will check if the is a bug, or the Unit Tests need to be updated.
I'm using my own decoding Hebrew map because Masav uses their own encoding.
This is a link to the mapping [link](https://www.masav.co.il/media/2001/hebrew.pdf)
Masav specification [link](https://www.masav.co.il/ts_download)

Thanks

**Comment by ElishaMayer** [comment]
Hi, I fixed the test in version 1.0.13.
The tests were not up to date with the implementation of "Masav" Hebrew encoding.
The files are tested with Masav and they show names in Hebrew correctly.
